src/util.rs

```diff
@@ -21,6 +24,41 @@ pub const BITS_PER_BYTE: u32 = 8;
 /// that are not valid in Rust ident
 const BLACKLIST_CHARS: &[char] = &['(', ')', '[', ']', '/', ' ', '-'];
 
+fn to_pascal_case(s: &str) -> String {
+    if !s.contains('_') {
+        s.to_pascal_case()
+    } else {
+        let mut string = String::new();
+        let mut parts = s.split('_').peekable();
```

Comment:
Should this maybe first convert to UPPER_CASE, and then convert to this "special" pascal case?

Reply:
Sorry, missed this in #805

Reply:
For `_1token` case? Maybe.

Reply:
No. Same result.

Reply:
I don't see other issues except this. And this name does not look correct for ident.
It's better to ping someone from ESP to fix SVD.

Reply:
ah, I was mistaken about what the code does, this looks fine